apps/site/app/docs.server.ts

```diff
@@ -88,14 +88,17 @@ export async function getManifest(
   return JSON.parse(manifest);
 }
 
-function getNodeText(node: RenderableTreeNode) {
+function getNodeText(renderableNode: RenderableTreeNode) {
+  if (typeof renderableNode === "string") return renderableNode;
```

Comment:
@jplhomer heads up that this line is a change in behavior. in the previous version of this function, if `typeof node === "string"`, the function returned an empty string, which i thought might be a bug, but if it is in fact desired, i will just lump this conditional in with the next when (line 93),